server/db/migrations/mysql/000001_create_calls_channels.up.sql

```diff
@@ -0,0 +1,33 @@
+CREATE TABLE IF NOT EXISTS CallsChannels (
+    ChannelID varchar(26) NOT NULL,
+    Enabled BOOLEAN,
+		Props JSON,
+    PRIMARY KEY (ChannelId)
+) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4;
+
+CREATE PROCEDURE MigrateCallsChannels ()
+BEGIN DECLARE
+	CallsChannels_Count INT;
+	SELECT
+		COUNT(*)
+	FROM
+		CallsChannels INTO CallsChannels_Count;
+	IF(CallsChannels_Count = 0) THEN
+		INSERT INTO CallsChannels (ChannelId, Enabled)
+		SELECT
+			PKey, JSON_EXTRACT(CONVERT(PValue using utf8mb4), "$.enabled") = true 
+		FROM
+			PluginKeyValueStore
+		WHERE
+			PluginId = 'com.mattermost.calls'
+		AND
+			LENGTH(PKey) = 26
+		AND
+			JSON_TYPE(JSON_EXTRACT(CONVERT(PValue using utf8mb4), "$.enabled")) != 'NULL';
```

Comment:
@cpoile As mentioned earlier today, if we wanted to be strict we would JOIN with Channels here to only insert values for non-archived/deleted channels although it makes this migration more complex with little benefit in my mind but let me know if you think otherwise.

Reply:
This seems fine to me -- avoiding joins with Channels is good, imo.